### ISSUE TITLE
auto-retalliation block all actions

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#473]
https://stat.tangaria.com/chars/Elendor-10340212102021.txt
player reported that he wasn't be able to drink potions. I suppose it's an old issue with auto-retalliation in melee which blocks all actions and player can not use items.. There is old solution to inscribe weapons with ^O, but new playres don't know it and get into this kinda unfair situation... so playing ranged class (archer or caster) by default player will 100% die cause of this problem and may die even several times until (if he won't quit the game till then cause of frustration which is quite probable) he reads game FAQ with:
![image](https://user-images.githubusercontent.com/39904162/136919554-181be2d2-603d-4a35-aa4a-20eb7e258b6c.png)
Which is pretty complex for new player to comprehend.

I wonder, maybe it's possible to make a hack which will disable auto-retalliation for _next_ turn if player use items during fight? eg:
1) player auto-retalliate
2) if player push drink potion - this action added to the queue
3) next turn: potion drinked
4) potions auto-retalliate again

**Comment by draconisPW** [comment]
Dump doesn't mention any auto relatiation though, it's just a player shooting arrows and getting killed by Orfax. You would have to check log prior to this, but i'm pretty sure it has nothing to do with auto retaliation. Maybe a mistake in the macro for potions?

**Comment by igroglaz** [comment]
..Though he said that he always played like this and it happens for the first time. It's not first 1st archer.

**Comment by igroglaz** [comment]
Indeed, he is quite experienced player: his past archer achieved lvl 41: https://stat.tangaria.com/chars/Arindil-16391904102021.txt

**Comment by igroglaz** [comment]
It may be just lag though..
